pkgs/desktops/plasma-5/plasma-bigscreen.nix

```diff
@@ -0,0 +1,48 @@
+{ mkDerivation
+, extra-cmake-modules
+, kactivities
+, kactivities-stats
+, plasma-framework
+, ki18n
+, kirigami2
+, kdeclarative
+, kcmutils
+, knotifications
+, kio
+, kwayland
+, kwindowsystem
+, plasma-workspace
+, qtmultimedia
+,
+}:
+mkDerivation {
+  pname = "plasma-bigscreen";
+  nativeBuildInputs = [ extra-cmake-modules ];
+  buildInputs = [
+    kactivities
+    kactivities-stats
+    plasma-framework
+    ki18n
+    kirigami2
+    kdeclarative
+    kcmutils
+    knotifications
+    kio
+    kwayland
+    kwindowsystem
+    plasma-workspace
+    qtmultimedia
+  ];
+
+  postPatch = ''
+    substituteInPlace bin/plasma-bigscreen-wayland.in \
+      --replace @KDE_INSTALL_FULL_LIBEXECDIR@ "${plasma-workspace}/libexec"
+  '';
+
+  preFixup = ''
+    wrapQtApp $out/bin/plasma-bigscreen-x11
+    wrapQtApp $out/bin/plasma-bigscreen-wayland
```

Comment:
You are sure there is no double wrapping happening?

Reply:
Yes, they're shell scripts.